bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs

```diff
@@ -32,12 +32,11 @@ parser.add_argument("--use-maxalpha", action="store_true")
 parser.add_argument("--coinc-threshold", type=float, default=0.0,
                     help="Seconds to add to time-of-flight coincidence window")
 parser.add_argument("--timeslide-interval", type=float,
-                    help="Interval between timeslides in seconds. Timeslides are"
-                         " disabled if the option is omitted.")
-parser.add_argument("--loudest-keep-values", type=str, nargs='*', default='6:1',
-                    help="Apply successive multiplicative levels of decimation"
-                         " to coincs with stat value below the given thresholds"
-                         " Ex. 9:10 8.5:30 8:30 7.5:30. Default: no decimation")
+                    help="Interval between timeslides in seconds. Timeslides are "
```

Comment:
This seems to be a revert of a change now on master. Can you rebase to upstream/master and push -f